source/MaterialXView/CMakeLists.txt

```diff
@@ -74,6 +74,9 @@ endif()
 if (MATERIALX_BUILD_GEN_MDL)
     LIST(APPEND LIBS MaterialXGenMdl)
 endif()
+if (MATERIALX_BUILD_GEN_ARNOLD)
+    LIST(APPEND LIBS MaterialXGenArnold)
```

Comment:
Just a note that MaterialXGenArnold is not part of Lucasfilm master. So should we avoid leaking use of that library? Or can we keep this apart when syncing with their master?

Reply:
We already have branching in a number of places so we keep this apart and never merge our stuff into master.